### DISCUSSION TITLE
loading jsdos savestates

### DISCUSSION THREAD

**Discussion by rockinsect** [DustinBrett/daedalOS#425] [Q&A] [answered]
i can't figure out for the life of me how to load the save states in /Users/Public/Snapshots into jsdos, how would i do that?
this website has been really useful in the short while i've been using it and is probably the coolest site i've seen, but that's the one thing i have yet to figure out that would make it 10/10

**Comment by DustinBrett** [comment, marked as answer]
Thanks for checking out my site! It currently loads automatically based on filename I believe. Do if the file is doom.jsdos then it should try and load the doom.jsdos.zip.save. For v86 it will start right back where the snapshot was. For jsdos it loads the same file state but you need to have saved the game and to load it within the game itself. So for Doom you would save from the menu, then you could come back to the page later and open Doom and load that same save from within Doom.

**Reply by rockinsect** [reply]
thank you, i guess the game i was trying to play didn't save its file correctly the first time so it didn't exist when i played again even though the snapshot worked

**Reply by DustinBrett** [reply]
Ah ya the snapshot creation is automatic but actually going into the game and making a game save which can be later loaded is typically not manual, such as for Doom.
